dev/vultr-cli.ts

```diff
@@ -0,0 +1,562 @@
+                {
+                  name: "appID",
+                  description: "Application ID",
+                },
+              ],
+            },
+            {
+              name: "list",
+              description: "Available apps a bare metal server can change to",
+              args: bareMetalIdArg,
+            },
+          ],
+        },
+        {
+          name: ["bandwidth", "b"],
+          description: "Get a bare metal server's bandwidth usage",
+          args: bareMetalIdArg,
+        },
+        {
+          name: ["create", "c"],
+          description: "Create a bare metal server",
+          options: createOptions,
+        },
+        {
+          name: ["delete", "destroy"],
+          description: "Delete a bare metal server",
+          isDangerous: true,
+          args: bareMetalIdArg,
+        },
+        {
+          name: "get",
+          description: "Get a bare metal server by <bareMetalID>",
+          args: bareMetalIdArg,
+        },
+        {
+          name: ["halt", "h"],
+          description: "Halt a bare metal server",
+          args: bareMetalIdArg,
+        },
+        {
+          name: ["image", "i"],
+          description:
+            "Image is used to access bare metal server image commands",
+          subcommands: [
+            {
+              name: "change",
+              description: "Change a bare metal server's image",
+              args: [
+                bareMetalIdArg,
+                {
+                  name: "imageID",
+                  description: "Image ID",
+                },
+              ],
+            },
+            {
+              name: "list",
+              description: "Available images a bare metal server can change to",
+              args: bareMetalIdArg,
+            },
+          ],
+        },
+        {
+          name: "ipv4",
+          description: "List the IPv4 information of a bare metal server",
+        },
+        {
+          name: "ipv6",
+          description: "List the IPv6 information of a bare metal server",
+        },
+        { name: "list", description: "List all bare metal servers" },
+        {
+          name: "os",
+          description:
+            "Os is used to access bare metal server operating system commands",
+        },
+        {
+          name: "reboot",
+          description:
+            "Reboot a bare metal server. This is a hard reboot, which means that the server is powered off, then back on",
+        },
+        {
+          name: "reinstall",
+          description: "Reinstall the operating system on a bare metal server",
+        },
+        { name: "start", description: "Start a bare metal server" },
+        {
+          name: "user-data",
+          description:
+            "User-data is used to access bare metal server user-data commands",
+        },
+        {
+          name: "vnc",
+          description: "Get a bare metal server's VNC url by <bareMetalID>",
```

Comment:
Does it accept a `bareMetalId` arg?
```suggestion
          description: "Get a bare metal server's VNC url by <bareMetalID>",
          args: bareMetalIdArg,
```